src/bcc_sensor.c

```diff
@@ -399,6 +409,48 @@ static inline int __do_file_path(struct pt_regs *ctx,
     return 0;
 }
 
+static inline int __do_dentry_path(struct pt_regs *ctx, struct dentry *dentry, struct data_t *data)
```

Comment:
To prevent iterating passed the root filesystem's mountpoint, we probably want to do a siimilar check __do_file_path does with `root_fs_dentry`. 

Filesystems like btrfs can run into these issues and you may see paths that can't be resolved.

If needed I can add this in another pull request.

Reply:
Let's do that as another pull.